assets/airports/eidw.json

```diff
@@ -883,124 +1076,361 @@
             "end":    [["N53d26m13.16", "W6d15m43.12", "217ft"], ["N53d25m11.66", "W6d14m58.54", "202ft"]],
             "length":      2.072,
             "ils":         [true, true]
+        },
+        {
+            "name":        ["10X", "28X"],
+            "name_offset": [[1000, 1000], [1000, 1000]],
+            "end":    [["N53d25m20.75", "W6d17m24.27", "242ft"], ["N53d25m12.94", "W6d15m02.08", "202ft"]],
+            "length":      2.637,
+            "ils":         [true, true]
+        },
+        {
+            "name":        ["16X", "34X"],
+            "name_offset": [[1000, 1000], [1000, 1000]],
+            "end":    [["N53d26m13.16", "W6d15m43.12", "217ft"], ["N53d25m11.66", "W6d14m58.54", "202ft"]],
+            "length":      2.072,
+            "ils":         [true, true]
         }
     ],
     "sids": {
```

Comment:
The restricted areas aren't active 24/7 actually, so technically they could be used. But in the 3+ years that I've been living about 30 minutes from EIDW, I've never seen an aircraft actually fly this departure (Doesn't mean it doesn't happen, I've just never witnessed it). I'll add it anyway since it's not much work.